### PR TITLE
issue #56 modificato link

### PR DESCRIPTION
**Pull request by SuperMauro1966**
----- ISSUE: #56

web/language-tools-and-standard-library/protocols/

corrected the link in the anchor HTTP GET/POST Requests

### CODE REVIEW

File: web/language-tools-and-standard-library/protocols/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ In this section we explain what parameters can be set for each protocol.
 
 The parameters of the protocol are specified in the input/output [port](https://jolielang.gitbook.io/docs/basics/communication-ports) definition. Unless required, if a parameter is not defined it is set to its default value according to its protocol specification.
 
-Let us recall the examples given in [HTTP GET/POST Requests](https://jolielang.gitbook.io/docs/web-applications/web_get_post) where we set the parameters `method` and `format`, of protocol `http`, to define what kind of messages the port shall send or receive.
+Let us recall the examples given in [HTTP GET/POST Requests](https://docs.jolie-lang.org/v1.10.x/language-tools-and-standard-library/web-applications/web_get_post.html) where we set the parameters `method` and `format`, of protocol `http`, to define what kind of messages the port shall send or receive.
```
Comment (fmontesi) [inline web/language-tools-and-standard-library/protocols/README.md:24]: Shouldn't we use relative URLs? @thesave @bmaschio what do you think?

Comment (thesave) [reply web/language-tools-and-standard-library/protocols/README.md:24]: Yes, it should be `../web-applications/web_get_post.md` IIRC.

**Comment by fmontesi** [comment]
@klag and @thesave this looks almost done..? Can you resolve?

**Comment by thesave** [comment]
Can we (reviewers) edit the original pull request?
I looked a bit into the PR interface but I can't figure out how to do that.
Anyhow, @SuperMauro1966 if you could kindly modify your original PR according to the comment, we should be fine and we can merge it.

**Comment by fmontesi** [comment]
@thesave do you see this?
Screenshot_20210518_151712

**Comment by thesave** [comment]
Yes, but my "edit" and "delete" are grayed out 😢
Screenshot 2021-05-18 at 17 35 48

**Pull request merged**